runtime-sdk/modules/contracts/src/store.rs

```diff
@@ -13,12 +13,26 @@ use crate::{state, types, Error, MODULE_NAME};
 pub const CONFIDENTIAL_STORE_KEY_PAIR_ID_CONTEXT_BASE: &[u8] =
     b"oasis-runtime-sdk/contracts: state";
 
+const CONTEXT_CONFIDENTIAL_STORE_TX_INSTANCE_COUNT: &str = "oasis-runtime-sdk/Context store instance count";
```

Comment:
If we have this do we even need the tx index given that this will already be per-block and will increment on each access?